third_party/opencv/exif/exif.cc

```diff
@@ -0,0 +1,553 @@
+// Copyright (c) 2022, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
I'm not sure if we can add our license on top. I would at most extend the copyright of the original file.

Reply:
If it is more than just a copy paste then it should be fine to do it this way.

Reply:
The libtiff decoder has been substantially altered. Here, we're barely introducing some mechanical changes. I think the changes we've introduced don't warrant sublicensing.

Reply:
In that case, I will remove our license blurb.